<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>callWindow</class>
 <widget class="QWidget" name="callWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>语音通话</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="acceptButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>接听</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rejectButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
    <widget class="QLabel" name="incomingCallLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>正在处理来电</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="inCallStatusLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>正在通话中</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hangupButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>挂断</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
